### DISCUSSION TITLE
Can't access console logs in Versel when using Next.js

### DISCUSSION THREAD

**Discussion by faresd** [vercel/next.js#15693] [Help]
# Bug report

## To Reproduce
The usage of
console.log
console.error
etc..

Don't get pushed to the deployed application realtime logs and I found no way to send them except if I use throw("my error")

Steps to reproduce the behavior, please provide code snippets or a repository:

console.log in you Next.js app and deploy to Vercel

## Expected behavior
send logs to the functions logs console.

## System information

- OS: [e.g. macOS, Windows]
- Browser (if applies) [e.g. chrome, safari]
- Version of Next.js: [e.g. 6.0.2]
- Version of Node.js: [e.g. 10.10.0]

**Comment by paulogdm** [comment]
Hello,

Please send us a message on https://vercel.com/support/request so we can take a look a what went wrong.

**Comment by guylevin** [comment]
Having the same issue
